### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by MeenaFoerderer**

### CODE REVIEW

File: components/BookmarkIcons.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import styled from "styled-components";
 import { BsFillBookmarkFill, BsBookmark } from "react-icons/bs";
 
 const BookmarkActive = styled(BsFillBookmarkFill)`
-  font-size: 2.5em;
-  color: #2a384f;
+  font-size: 2em;
```
Comment (StefanFitoussi) [inline components/BookmarkIcons.js:5]: rem

File: components/BookmarkIcons.js [hunk 2]
```diff
@@ -2,13 +2,13 @@ import styled from "styled-components";
...
+  color: #493843;
 `;
 
 const BookmarkInactive = styled(BsBookmark)`
-  font-size: 2.5em;
-  color: #2a384f;
+  font-size: 2em;
```
Comment (StefanFitoussi) [inline components/BookmarkIcons.js:10]: rem

File: components/FooterElements.js [hunk 3]
```diff
@@ -9,14 +9,13 @@ import {
 } from "react-icons/bs";
 
 const FooterNav = styled.nav`
-  background-color: #2a384f;
+  background-color: #fff;
   box-shadow: rgba(0, 0, 0, 0.1) 0px 0px 5px 0px,
     rgba(0, 0, 0, 0.1) 0px 0px 1px 0px;
-  height: 4em;
+  height: 65px;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:15]: rem

File: components/FooterElements.js [hunk 4]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
-  color: #f5f5f5;
+  color: #7a7a7a;
+
+  &:active:focus {
+    color: #f9f9f9;
+  }
 `;
 
 const PrevRoomIcon = styled(BsArrowLeftSquare)`
   font-size: 2em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:51]: rem

File: components/FooterElements.js [hunk 5]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
-  color: #f5f5f5;
+  color: #7a7a7a;
+
+  &:active:focus {
+    color: #f9f9f9;
+  }
 `;
 
 const NextRoomIcon = styled(BsArrowRightSquare)`
   font-size: 2em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:60]: rrem

File: components/FooterElements.js [hunk 6]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
-  color: #f5f5f5;
+  color: #7a7a7a;
+
+  &:active:focus {
+    color: #f9f9f9;
+  }
 `;
 
 const HomeIcon = styled(BsHouseDoor)`
-  font-size: 2.5em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:52]: rem

File: components/FooterElements.js [hunk 7]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
-  color: #f5f5f5;
+  font-size: 2em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:69]: rem

File: components/FooterElements.js [hunk 8]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
+  color: #7a7a7a;
+
+  &:active:focus {
+    color: #f9f9f9;
+  }
 `;
 
 const BookmarkIcon = styled(BsJournalBookmarkFill)`
-  font-size: 2.5em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:57]: rem

File: components/FooterElements.js [hunk 9]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
-  color: #f5f5f5;
+  font-size: 2em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:78]: rem

File: components/FooterElements.js [hunk 10]
```diff
@@ -29,38 +28,68 @@ const FooterLink = styled(Link)`
...
+
+  &:active:focus {
+    color: #f9f9f9;
+  }
 `;
 
 const CalendarIcon = styled(BsCalendar4Week)`
-  font-size: 2.5em;
-  color: #f5f5f5;
+  font-size: 2em;
```
Comment (StefanFitoussi) [inline components/FooterElements.js:87]: rem

File: pages/[date]/[room]/talks/[id].js [hunk 11]
```diff
@@ -102,22 +103,28 @@ const IconWrapper = styled.div`
 const CloseIcon = styled(IoCloseCircleOutline)`
   font-size: 2.5em;
```
Comment (StefanFitoussi) [inline pages/[date]/[room]/talks/[id].js:104]: rem

**Comment by MeenaFoerderer** [comment]
Thank you Stefan! Sadly, it didn't solve my problem with the varying footer height 😶

**Pull request merged**